src/components/_materialUi/Select/index.stories.tsx

```diff
@@ -20,6 +20,7 @@ import { Select } from './internal';
 export default {
   args: {
     children: Array(...Array(10)).map((_, i): any => (
+      // eslint-disable-next-line react/no-array-index-key
```

Comment:
If you import range from lodash, this would be a lot cleaner as:

(Function return type to be validated. I'm allergic to `any`.

```suggestion
    children: range(10).map((i): React.ReactElement<unknown> => (
```

Reply:
Nice! Will refactor that